### PR TITLE
community: add tencent cloud as an adopter

### PR DESCRIPTION
**Pull request by Xunzhuo**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Release Notes: No

### CODE REVIEW

**Review by arkodg — APPROVED** [review]
LGTM thanks !

**Review by zhaohuabing — APPROVED** [review]
LGTM thanks @Xunzhuo for making this happen!

**Comment by arkodg** [comment]
this doesnt seem like the right place @Xunzhuo, we need to update https://github.com/envoyproxy/gateway/blob/main/site/data/adopters.yaml

**Review by arkodg — APPROVED** [review]

File: ADOPTERS.md [hunk 1]
```diff
@@ -43,6 +45,7 @@ If you would like to be included in this table, please submit a PR to this file
 * (Option) Description:
 
 ## Airspace Link
+
```
Comment (arkodg) [inline ADOPTERS.md:48]: should we delete the content here and only maintain `site/data/adopters.yaml` ?

Comment (Xunzhuo) [reply ADOPTERS.md:48]: LGTM

**Pull request merged**